backend/src/middleware/main.ts

```diff
@@ -49,6 +49,7 @@ export async function main(context: EventContext<Env, any, any>) {
 	if (
 		url.pathname === '/oauth/token' ||
 		url.pathname === '/oauth/authorize' || // Cloudflare Access runs on /oauth/authorize
+		url.pathname === '/api/v1/statuses' || // Unless private https://docs.joinmastodon.org/methods/statuses/#get
```

Comment:
Should be `url.pathname.startsWith('/api/v1/statuses/')` no?

Reply:
Ideally a regex to keep `/api/v1/statuses/[id]/reblog` or  `/api/v1/statuses/[id]/favourite` authentificated

Reply:
Makes sense, added the regex